.pre-commit-config.yaml

```diff
@@ -59,14 +52,4 @@ repos:
       exclude: |
         (?x)^(
         .*/__init__.py |
-        fv3gfs-physics/.*
-        )$
-    - id: flake8
-      name: flake8 __init__.py files
-      files: "__init__.py"
-      # ignore unused import error in __init__.py files
-      args: ["--ignore=F401,E203", --config, setup.cfg]
-      exclude: |
-        (?x)^(
-        fv3gfs-physics/.*
         )$
```

Comment:
What you'll need to do here is delete the fv3gfs-physics entry from the first `exclude` but leave the __init__.py one, and delete the `exclude` section altogether from the second entry. There should still be two flake8 entries.